docs/guides/cli.md

```diff
@@ -51,5 +52,8 @@ Connection: keep-alive
 
 The body, headers, etc. for this response will be taken from the API description document.
 
+## Running in Production
```

Comment:
Why would I run prism in production?

Reply:
People do all sorts of things and I'd rather have an example for them than not. I'm moving this from a nested README to a smarter place, and we can improve on its explanation another time.

Reply:
Prims will run in production on the hosted platform (Stoplight), for example. I want *you* to know how to use it properly :trollface: